great-idea-website/index.html

```diff
@@ -1,63 +1,109 @@
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+
+      <div class="opener">
+        <div class="openertext">
+          <h1>
+            <p>Innovation</p>
+            <p>On</p>
+            <p>Demand</p>
+          </h1>
+
+          <p><button>Get Started</button></p>
+        </div>
+
+        <div class="openerimg">
+          <img src="img/header-img.png" alt="Image of a code snippet.">
+        </div>
+      </div>
+    </header>
+
+    <div class="content">
+      <hr>
+      <section class="contentrow1">
+        <section class="contentblock">
+          <h3>Features</h3>
+          Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+        <section class="contentblock">
+          <h3>About</h3>
+          Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+      </section>
+
+
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+      <section class="contentrow2">
+        <section class="contentblock">
+          <h3>Services</h3>
+          Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+
+        <section class="contentblock">
+          <h3>Product</h3>
+          Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+
+        <section class="contentblock">
+          <h3>Vision</h3>
+          Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+      </section>
+      <hr>
+
+      <footer>
+        <section class="Contact">
+          <h3>Contact</h3>
+          <p>123 Way 456 Street</p>
+          <p>Somewhere, USA</p>
+          <p class="phone">1 (888) 888-8888</p>
+          <p class="email">sales@greatidea.io</p>
+        </section>
+
+        <section class="copyright">Copyright Great Idea! 2018</section>
+      </footer>
+    </div>
+  </div>
+</body>
 
-  Copyright Great Idea! 2018
-  
-  </body>
 </html>
```

Comment:
Overall May. you did an amazing job on your first project.  Again, amazing to see you were able to apply animations to the navigation.  There are some layout issues with the logo and the code image, but CSS positioning takes some time to master, especially without flexbox.